src/jackknife.jl

```diff
@@ -0,0 +1,163 @@
+"""
+    Delete 1 jackknife with  for unstratified designs.
+    
+    Replicate weights are given by:
+    wi(hj)  = 0, if observation unit i is in psu j of stratum h
+            = nh(nh −1)wi, if observation unit i is in stratum h but not in psu j.
+    ## Reference
+    pg 380-382, Section 9.3.2 Jackknife - Sharon Lohr, Sampling Design and Analysis (2010)
+"""
+function jk1weights(design::SurveyDesign)
+    cluster_sorted = sort(design.data, design.cluster) # Does jk need sorting?
+    psus = unique(cluster_sorted[!, design.cluster])
+    nh = length(psus)
+    npsus = [(count(==(i), cluster_sorted[!, design.cluster])) for i in psus]       
+    replicates = [filter(n -> n != i, 1:nh) for i in 1:nh]
+    cluster_weights = cluster_sorted[!, design.weights]
+    for (i,replicate) in enumerate(replicates)
+        cluster_sorted[!, "replicate_"*string(i)] = vcat(
+            [   fill((count(==(i), replicate)) .* (nh / (nh - 1)), npsus[i]) for i = 1:nh
```

Comment:
I think this works fine, but again for clarity of code: should we use `j` instead of `i` here? Since `i` is being used in the outer for loop as well?

Reply:
Also, shouldn't this be `nh * (nh - 1)` instead of `nh / (nh - 1)`?

Reply:
no the multiplier is `nh / (nh - 1)` here